cmd/kgo-verifier/main.go

```diff
@@ -88,7 +88,9 @@ func main() {
 	}
 
 	signalChan := make(chan os.Signal, 1)
-	signal.Notify(signalChan, os.Interrupt)
+	if *remote {
```

Comment:
The symptom is that kgo-verifier launched without `--remote` doesn't stop after Ctrl-C. I think this happens because `signal.Notify` redefines the signal handler so the only thing that happens is that a message is sent to `signalChan`. And if `*remote` is false, nobody is listening on the channel (line 249)